README.md

```diff
@@ -61,13 +34,13 @@ For more information on the importance of a professional README for your reposit
 - [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
-<!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+
+# 📖 [eHllo microverse!] <a name="Hello microverse!"></a>
 
 > Describe your project in 1 or 2 sentences.
```

Comment:
- [ ] Again as requested by the previous reviewer in Pr #1, kindly remove line 41 as this is a comment to explain ro us what we should add to this section. Its called a section description, and we need to remove it and add our own description of the section.

e,g
`Hello Microverse is a project that takes two files.......`